core/trino-parser/src/main/antlr4/io/trino/sql/parser/SqlBase.g4

```diff
@@ -55,11 +55,12 @@ statement
     | DROP SCHEMA (IF EXISTS)? qualifiedName (CASCADE | RESTRICT)?     #dropSchema
     | ALTER SCHEMA qualifiedName RENAME TO identifier                  #renameSchema
     | ALTER SCHEMA qualifiedName SET AUTHORIZATION principal           #setSchemaAuthorization
-    | CREATE TABLE (IF NOT EXISTS)? qualifiedName columnAliases?
+    | CREATE (OR REPLACE)? TABLE (IF NOT EXISTS)? qualifiedName
```

Comment:
The `OR REPLACE` clause is not in the SQL standard. Do we have a consensus we want to add this? cc @martint

Reply:
This was discussed in #13180